filetypes.md

```diff
@@ -0,0 +1,56 @@
+| application/vnd.openxmlformats-officedocument.spreadsheetml.template      | xls             | tika           |
+| application/vnd.ms-excel.sheet.macroEnabled.12                            | xls             | tika           |
+| application/vnd.ms-excel.template.macroEnabled.12                         | xls             | tika           |
+| application/vnd.ms-excel.addin.macroEnabled.12                            | xls             | tika           |
+| application/vnd.ms-excel.sheet.binary.macroEnabled.12                     | xls             | tika           |
+| application/vnd.oasis.opendocument.spreadsheet-template                   | xls             | tika           |
+| application/vnd.oasis.opendocument.spreadsheet                            | xls             | tika           |
+| application/vnd.openxmlformats-officedocument.presentationml.presentation | ppt             | tika           |
+| application/vnd.openxmlformats-officedocument.presentationml.template     | ppt             | tika           |
+| application/vnd.openxmlformats-officedocument.presentationml.slideshow    | ppt             | tika           |
+| application/vnd.ms-powerpoint                                             | ppt             | tika           |
+| application/vnd.ms-powerpoint.addin.macroEnabled.12                       | ppt             | tika           |
+| application/vnd.ms-powerpoint.presentation.macroEnabled.12                | ppt             | tika           |
+| application/vnd.ms-powerpoint.template.macroEnabled.12                    | ppt             | tika           |
+| application/vnd.ms-powerpoint.slideshow.macroEnabled.12                   | ptt             | tika           |
+| application/vnd.oasis.opendocument.presentation                           | ppt             | tika           |
+| application/vnd.oasis.opendocument.presentation-template                  | ppt             | tika           |
+| application/xhtml+xml                                                     | -               | html           |
+| application/xml                                                           | -               | html           |
+| text/xml                                                                  | -               | htlm           |
+| image/tiff                                                                | image           | exif, ocr      |
+| image/jpg                                                                 | image           | exif, ocr      |
+| image/webp                                                                | image           | exif, ocr      |
+| image/heic                                                                | image           | exif, ocr      |
+| application/vnd.ms-outlook                                                | email           | email          |
+| application/vnd.ms-office                                                 | -               | email          |
```

Comment:
i suppose `application/vnd.ms-office` and `application/CDFV2` are micro$ email formats -let's map them to `email` too